src/mattack_actors.h

```diff
@@ -76,11 +76,11 @@ class gun_actor : public mattack_actor
         /*@}*/
 
         /*@{*/
-        /** Additional caps above any imposed by either the base gun or monster */
-        int range = 18;
-        int range_no_burst = INT_MAX; /** only burst fire at or below this distance */
+        /** Additional caps above any imposed by either the base gun or monster. If unset, from gun+ammo. */
+        int target_range = -1;
```

Comment:
Why the name change? Also the comment is slightly unclear and would be better defined as *if zero or negative set from gun+ammo*